File: src/main/java/com/cle/onlinetestsystem/pojo/Student.java
package com.cle.onlinetestsystem.pojo;

import com.baomidou.mybatisplus.annotation.TableId;
import lombok.Data;

@Data
public class Student {
@TableId
  private Long studentId;//学生id
  private Long classId;//班级id
  private String studentNo;//学生编号
  private String password;//密码
  private String studentName;//学生姓名

}

File: src/utils/Student/Student.ts
export default class Student {
  public studentId: string = ''
  public studentNo: string = ''
  public studentName: string = ''
  public classNo: string = ''

  constructor(
    studentId: string,
    studentNo: string,
    studentName: string,
    classNo: string
  ) {
    this.studentId = studentId
    this.studentNo = studentNo
    this.studentName = studentName
    this.classNo = classNo
  }
}
